src/navigator/Navigator.tsx

```diff
@@ -736,7 +743,7 @@ function nativeBottomSheets(BottomSheet: typeof RootStack) {
 
 function ModalStackScreen() {
   return (
-    <ModalStack.Navigator mode="modal">
```

Comment:
Why remove?

Reply:
`mode="modal"` is removed in v6. Can take a look at some similar options: [docs](https://reactnavigation.org/docs/upgrading-from-5.x/#modemodal-is-removed-in-favor-of-presentation-modal).

Reply:
Can we switch it to `presentation: 'modal'` then?